backend/README.md

```diff
@@ -0,0 +1,36 @@
+# Eiffelvis Backend
+[![Backend CI](https://github.com/ItJustWorksTM/EiffelVis/actions/workflows/backend_ci.yml/badge.svg)](https://github.com/ItJustWorksTM/EiffelVis/actions/workflows/backend_ci.yml)
+
+Eiffelvis is mainly implemented inside [eiffelvis_core](./crates/eiffelvis_core), it makes up 90% of the actual code along with [eiffelvis_http](./crates/eiffelvis_http). This top level binary simply bundles all these into a usable executable with command line options.
```

Comment:
The second sentence is a bit weird. [...] bundles all these into [...]. Add either "crates" or "libraries" to make it clearer